.github/workflows/publish.yml

```diff
@@ -34,3 +34,22 @@ jobs:
       - name: Deploy TUF-on-CI repository to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@7a9bd943aa5e5175aeb8502edcc6c1c02d398e10 # v4.0.2
+
+  test:
+    needs: deploy
+    permissions:
+      issues: 'write' # for modifying Issues
```

Comment:
this is not actually used in our case but GHA requires it because the workflow does write to issues when triggered on schedule.